lambda/update/main_test.go

```diff
@@ -14,13 +14,32 @@ import (
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/google/uuid"
 	"github.com/golang-jwt/jwt/v5"
+	"github.com/ministryofjustice/opg-data-lpa-store/internal/event"
 	"github.com/ministryofjustice/opg-data-lpa-store/internal/shared"
 	"github.com/ministryofjustice/opg-go-common/logging"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 )
 
 var expectedError = errors.New("expected")
 
+type mockLogger struct {
+	mock.Mock
```

Comment:
One day when someone has free time, it might be nice to explore https://vektra.github.io/mockery/latest/. It's really simplified all of our tests on modernise.

Reply:
👍🏼  Thanks - that'd be a good one as part of my L&D